src/client/clientenvironment.cpp

```diff
@@ -222,7 +208,7 @@ void ClientEnvironment::step(float dtime)
 				v3f speed = lplayer->getSpeed();
 				if (!lplayer->in_liquid)
 					speed.Y -= lplayer->movement_gravity *
-						lplayer->physics_override_gravity * dtime_part * 2.0f;
+							lplayer->physics_override_gravity * dtime_part * 2.0f;
```

Comment:
I'd prefer indenting continuation lines by only one extra tab. Enforcing a 2-tab indent would result in lines reaching the line length limit quicker.

Reply:
I can't work out how to have only a single tab indent and also indent twice in if-statements. There's still a longer line limit after this PR

Reply:
This is what I do usually (one tab) - two tabs seems like too much, agreed with ClobberXD.

Reply:
It needs to either be 2 tabs or no tabs. If you don't put your starting brace on its own line 1 tab will cause confusion

Reply:
Ah, in this case I guess you're stuck unless you can figure out how to format it seperately

Reply:
That is horrible, conditions should use braces if the action line is long enough to need continuation lines.

Reply:
We usually omit braces only for single lines, but not for single statements spanning multiple lines. If this isn't configurable in clang-format, I'd rather prefer to use braces everywhere.